scripts/filemanager.js

```diff
@@ -2259,61 +2319,73 @@ $(function() {
 	loadJS('/scripts/zeroclipboard/dist/ZeroClipboard.js');
 
 	// Loading CodeMirror if enabled for online edition
-	if(config.edit.enabled) {
+	if(fm.config.edit.enabled) {
 		loadCSS('/scripts/CodeMirror/lib/codemirror.css');
-		loadCSS('/scripts/CodeMirror/theme/' + config.edit.theme + '.css');
+		loadCSS('/scripts/CodeMirror/theme/' + fm.config.edit.theme + '.css');
 		loadJS('/scripts/CodeMirror/lib/codemirror.js');
 		loadJS('/scripts/CodeMirror/addon/selection/active-line.js');
 		loadCSS('/scripts/CodeMirror/addon/display/fullscreen.css');
 		loadJS('/scripts/CodeMirror/addon/display/fullscreen.js');
 		loadJS('/scripts/CodeMirror/dynamic-mode.js');
 	}
+  
+  // added gk from simogeo, why was it removed?
```

Comment:
Yes, you could pull new changes from master.
Also read [this topic](https://github.com/servocoder/RichFilemanager/issues/18), please. It's about new features of "Preview" and "Thumbnail" params in response to "getinfo" request. The solution, that we came to, is implemented and is on master.